src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        List<Truck> trucks = new ArrayList<>();
+        trucks.add(new Truck());
+        trucks.add(new Truck());
+        return trucks;
```

Comment:
Probably we want to create _all-args_ constructors for out models and populate their fields with some values